src/app/wallets/intraledger-send-payment.ts

```diff
@@ -55,6 +55,9 @@ export const intraledgerPaymentSendWalletId = async ({
 
   if (senderWalletId === recipientWalletId) return new SelfPaymentError()
 
+  const recipientWallet = await WalletsRepository().findById(recipientWalletId)
```

Comment:
where is this value is used?

Reply:
Only being used below to check if the wallet exists or not